packages/components/popover/src/popover.ts

```diff
@@ -14,20 +14,28 @@ let nextUniqueId = 0;
  * @slot default - Body content for the popover
  */
 export class Popover extends LitElement {
+  /** The default padding of the arrow. */
+  static arrowPadding = 8;
```

Comment:
Maybe the arrowPadding should be 16?

<img width="551" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/5e8e981d-10e1-4b39-adf4-cf8a3748b356">

Reply:
https://www.figma.com/file/1oIVcD9VMbOtLXYp48ehsA/Sanoma-Learning-Light-mode?type=design&node-id=5105-7255&mode=dev